docs/change-log.md

```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update December 2nd, 2022
+
+- **New:** Released **Pausing & Resuming of Environments** feature which allows you to scale down all of the Workloads of type Deployment to 0 replicas with one click of a button or single API call. Resuming the environment will scale the Workloads back to the number of replicas that was set at the time of Pausing. This can help you save costs on unused infrastructure. You can find more detailed information on a dedicated [Docs page](https://docs.humanitec.com/using-humanitec/work-with-environments/pause-and-resume).
```

Comment:
- "number of replicas that was set": should be that "were" set if I'm not mistaken.
- "at the time of Pausing": is "Pausing" capitalised here on purpose?
- maybe just "more information in our documentation"?